group.go

```diff
@@ -34,9 +36,19 @@ type Group struct {
 	lenC chan int
 }
 
+type GroupOption func(*Group)
```

Comment:
Note the package prefix: we'll get `schedgroup. GroupOption`. IMO `schedgroup .Option` is better choice

Reply:
Plus any comment like: `// Option allows to tune Group behaviour on creation.`